tests/keras/layers/noise_test.py

```diff
@@ -23,5 +23,14 @@ def test_GaussianDropout():
                input_shape=(3, 2, 3))
 
 
+@keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason="cntk does not support it yet")
```

Comment:
I mean to replace "cntk does not support it yet" with eg "cntk does not support AlphaDropout yet"
This way whenever I see this notice I can check whether or not it is still the case - and hopefully test will be unskipped as soon as CNTK comes up with necessary feature